SoftLayer/managers/block.py

```diff
@@ -510,16 +510,20 @@ def cancel_block_volume(self, volume_id,
         block_volume = self.get_block_volume_details(
             volume_id,
             mask='mask[id,billingItem[id,hourlyFlag]]')
-        billing_item_id = block_volume['billingItem']['id']
+        billingItem = 'billingItem' in block_volume
+        if billingItem is False:
+            print('The block storage was cancelled')
+        else:
```

Comment:
Since were going to throw an exception if we don't have a billing item, the else bits are not really needed and can be reverted back.